lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -13,6 +13,13 @@
 # the building of the a.out executable program.  For example,
 # DYLIB_ONLY := YES
 #
+# When specifying one of the DYLIB_*_SOURCES variables, DYLIB_NAME
+# controls the name of the produced dylib. E.g., if set to "foo",
+# the generated dylib will be called "foo.<platform-specific-extension>",
+# which on Darwin will be "foo.dylib".
```

Comment:
This is interesting. LLDB makes an assumption that a module named `foo` will be turned onto `libfoo.dylib` on Darwin. See `Platform::GetFullNameForDylib`

Reply:
good catch! it was meant to be `libfoo` for the darwin example :)

Reply:
Technically, the "lib" part is platform specific as well. Windows does not have that convention. Libraries (at least, those that aren't very obviously unixy in origin) don't have it: https://github.com/llvm/llvm-project/blob/5a09ce9e038ed73ea60edf5463dd6509f7c4848f/lldb/packages/Python/lldbsuite/test/lldbplatformutil.py#L212

Reply:
Yup, reworded to make it clear this is platform dependent (with an example of how it would look like on Darwin)